otter/integration/config.sh-template

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# To run the integration tests, you'll need to have various environment
+# settings established.
+
+# This is the username used to authenticate against Identity.
+export AS_USERNAME=joe_user
+
+# This is the password used to authenticate against Identity.
+export AS_PASSWORD=joes_password
```

Comment:
OK, I'm stupid.  I see now.  Edit said file to look for configuration scripts and detect instances of AS_PASSWORD and AS_USERNAME and such.  Got it.  That's a good idea.  I'm guessing this runs as a pre-commit hook on github?  Or is this something we run manually?